blog/config/common/routes/routes.php

```diff
@@ -36,11 +37,22 @@
 use Yiisoft\Yii\RateLimiter\LimitRequestsMiddleware;
 use Yiisoft\Yii\RateLimiter\Storage\StorageInterface;
 
+$routes = [];
+foreach (Attributes::findTargetMethods(Route::class) as $method) {
```

Comment:
What about classes and groups?

Reply:
It's just an example. If we want to merge it I'll migrate all routes and add groups and classes